core/python2Action/pythonrunner.py

```diff
@@ -0,0 +1,87 @@
+"""Executable Python script for running Python actions.
```

Comment:
Fixed in https://github.com/openwhisk/openwhisk/pull/1903/files#diff-01c3377ed05fef923a1cc0d1b2257e1dR39